indico/modules/search/client/js/components/results/Event.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import {List} from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import './Event.module.scss';
+import moment from 'moment';
+import CategoryPath from './CategoryPath';
+
+const Event = ({
+  // id,
+  type,
+  title,
+  url,
+  // description,
+  categoryPath,
+  startDt,
+  endDt,
+  // location,
+  speakers,
+  // chairs,
+}) => {
+  const singleDay =
+    moment(startDt, 'YYYY-MM-DDZhh:mm').format('ll') ===
+    moment(endDt, 'YYYY-MM-DDZhh:mm').format('ll');
+
+  return (
+    <div styleName="event">
+      <List.Header>
+        <a href={url}>{title}</a>
+      </List.Header>
+      <List.Description styleName="description">
+        {/* if it's a lecture print the list of speakers */}
+        {type === 'lecture' && speakers.length !== 0 && (
+          <List.Item styleName="high-priority">
+            {speakers.map(i => (
+              <div key={i.name}>
+                {i.title
+                  ? `${i.title} ${i.name} (${i.affiliation})`
+                  : `${i.name} (${i.affiliation})`}
+              </div>
+            ))}
+          </List.Item>
+        )}
+
+        {/* Dates */}
+        {/* if end date == start date only show start date */}
+        {singleDay ? (
```

Comment:
Maybe the date rendering part could be extracted out to a function, especially if we reuse it elsewhere.